### DISCUSSION TITLE
Using @persist('scrollbar')  results in AlpineJS error

### DISCUSSION THREAD

**Discussion by francoism90** [livewire/livewire#8396] [7. Bugs (archived)]
### Livewire version

3.4.11

### Laravel version

11.6.0

### Which PHP version are you using?

PHP 8.3

### Steps To Reproduce

I'm trying to build a full viewport/window 'snap-y' feed.

This is a part of the blade I've created:

```blade
@persist('scrollbar')
    <div
        class="relative h-[calc(100dvh)] min-h-[calc(100dvh)] overflow-y-scroll snap-y snap-mandatory"
        wire:scroll
    >
        @foreach ($this->items as $item)
            <x-app.item :$item />
        @endforeach

        <div x-intersect.full="$wire.fetch()"></div>
    </div>
@endpersist
```

However this results in the following error message:

![image](https://github.com/livewire/livewire/assets/5028905/e5b6dd31-b757-44ae-865a-91ad9a92ab59)

I think it's something Livewire specific.

Just to clarify, it all works fine. The error goes away when remove `overflow-y-scroll`, however this prevents the scroll from working.

I don't know if this is an user problem or a bug in Livewire itself. It seems to attach to the incorrect $wire target?

### Repository or Wirebox URL

https://github.com/francoism90/hub/tree/feat-panel

### Additional information

Index: https://github.com/francoism90/hub/blob/feat-panel/resources/views/livewire/app/pages/feed/index.blade.php

Layout: https://github.com/francoism90/hub/blob/feat-panel/resources/views/components/layouts/simple.blade.php

The overlay is hidden for the body.

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by alxndrmlr** [comment]
I think this is also related to https://github.com/livewire/livewire/discussions/8217

**Reply by alxndrmlr** [reply]
ie: something going on with h-full

**Comment by pikseli** [comment]
I can confirm this happens with `wire:scroll` inside `@persist` or `x-persist` when the height of the content is larger than the height of the container (a scrollbar is needed) and you start to scroll the element.

**Reply by francoism90** [reply]
@pikseli Would this mean it's an AlpineJS bug, and not a Livewire one?

**Reply by pikseli** [reply]
I'm not familiar with the internals, but [/js/directives/wire-wildcard.js line 22](https://github.com/livewire/livewire/blob/b818a5b85f5ee041b6568fea16a2926a7b91c193/js/directives/wire-wildcard.js#L22) seems to be the culprit for this error. Apparently, directive.expression has no value.
